hoodie-common/src/main/java/com/uber/hoodie/common/model/FileSlice.java

```diff
@@ -81,10 +99,26 @@ public String getFileId() {
     return Optional.ofNullable(dataFile);
   }
 
+  public Optional<String> getOutstandingCompactionInstant() {
+    return Optional.ofNullable(outstandingCompactionInstant);
+  }
+
+  public void setOutstandingCompactionInstant(String outstandingCompactionInstant) {
+    this.outstandingCompactionInstant = outstandingCompactionInstant;
+  }
+
+  public String getBaseInstantForLogAppend() {
```

Comment:
So basically, when inserts into log files are done for the first time, there will be no fileSlice and we will create a new file. But the second time this will return the baseCommitTime for the first written log file even if there was no compaction scheduled, correct ?